data/explorer/beaconMetadata.json

```diff
@@ -2,12 +2,8 @@
   "0x09a5873667837598bd0990ba2f53d750d545ce435ecdcd44e0b4c64ab7d7d20d": {
     "category": "Cryptocurrency",
     "pricingCoverage": {
-      "polygon": "polygon",
```

Comment:
Why are you deleting these in this PR? Why did they exist before? Are you sure this doesn't break the Market?

Reply:
It will be okay for the market to remove this value, as long as this specific beacon doesn't belong to the polygon network.

My suggestion would be to keep it, as its presence doesn't hurt either. 

The reason in favor of keeping it is consistency. I've noticed that all beacons are sharing this same `pricingCoverage` block (counted 52 occurrences), which only does a mapping between the chain name and a pricing coverage id. 

I think there's a lot of redundant data. If you want to extract only one occurrence of it and move it to `explorer.pricingCoverage` or in some other place, while deleting the rest, the market can easily adapt to this.

```ts
  "pricingCoverage": {
      "polygon": "polygon",
      "polygon-testnet": "test-pricing-set-free",
      "rsk": "rsk",
      "avalanche": "avalanche",
      "avalanche-testnet": "test-pricing-set-free",
      "bsc": "bsc"
  }
```

Reply:
Doesn't `beaconSetMetadata.json` also have redundant chains listed under `pricingCoverage`? Removing these only in `beaconMetadata.json` doesn't feel consistent.

Reply:
Both files now have the same validation and there should not be redundant `pricingCoverage` data. The 4 `beaconSets` that I added have the same `chains` as the underlying `beacon` (since the idea was to have a single `beacon` `beaconSet` for each standard token BTC, ETH, USDC and USDT) https://app.slack.com/client/T0104044FT5/C03GPUX7P71

Reply:
Regarding the `pricingCoverage` linkage:
The original idea behind this was to have the ability to have different pricing on different chains but also potentially different pricing for different feeds on different chains.

The Market is possibly unaffected by this because the original developers probably took a shortcut ;)

Currently the full flexibility of this system isn't used because all our feeds are free on all chains.